src/wallet/rpcwallet.cpp

```diff
@@ -1098,7 +1098,28 @@ UniValue sendmany(const JSONRPCRequest& request)
 }
 
 // Defined in rpc/misc.cpp
-extern CScript _createmultisig_redeemScript(CWallet * const pwallet, const UniValue& params);
+extern CScript _createmultisig_redeemScript(const int required, const std::vector<CPubKey>& pubkeys);
```

Comment:
nit: really sucks that these are not put in headers - we're not just waiting to remove them in 0.17 when the deprecation goes away, but we're stuck with them. Can we find some place to put them?

Reply:
> nit: really sucks that these are not put in headers - we're not just waiting to remove them in 0.17 when the deprecation goes away, but we're stuck with them. Can we find some place to put them?

Agree these are really ugly. I would put them in rpc/util.h and rpc/util.cpp files, where other rpc helper functions like ParseConfirmTarget could live as well. As mentioned above, I would also change these to normal CamelCase, drop mention of multisig in _createmultisig_hex_to_pubkey and _createmultisig_addr_to_pubkey since they are not multisig-specific, and change the wallet argument to a keystore pointer in the latter since it is not wallet-specific.

Reply:
I created `rpc/util.cpp` and `rpc/util.h` and moved the functions there. I also renamed them to be `HexToPubKey` and `AddrToPubKey`.